ckanext/subscribe/action.py

```diff
@@ -20,6 +21,23 @@
 _check_access = p.toolkit.check_access
 NotFound = p.toolkit.ObjectNotFound
 
+CAPTCHA_API_URL = 'https://www.google.com/recaptcha/api/siteverify'
+
+def _verify_recaptcha(recaptcha_response):
+    secret_key = tk.config.get('ckan.recaptcha.privatekey', '')
```

Comment:
Could you please add the recaptcha stuff to the ckanext config so that users can choose whether or not to set it up for their sites? It would be nice to have it like this:

- the recaptcha config settings are specific to this extension, e.g. `ckanext.subscribe.recaptcha.api_url` and similar for the private key
- the settings are documented the readme (https://github.com/bellisk/ckanext-subscribe?tab=readme-ov-file#config-settings)
- the extension works without setting up reCAPTCHA